test/LifToken.js

```diff
@@ -79,9 +79,10 @@ contract('LifToken', function(accounts) {
 
     var dataEncoded = help.hexEncode(JSON.stringify({awesomeField:"AwesomeString"}));
     let transaction = await token.transferData(accounts[2], help.formatBalance(1000), dataEncoded, false, {from: accounts[1]});
+    let decodedEvents = help.abiDecoder.decodeLogs(transaction.receipt.logs);
```

Comment:
We execute calls inside our contracts, this are internal calls that can trigger events too. The events on the first transaction and function called are on transaction.logs, and the events triggered on internal functions are on transaction.receipt.logs. So if you want to get all the events you should look at transaction.receipt.logs, the bad thing is that here the events are encoded, but we decode them using abi-decoder lib